<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaysCalculator</class>
 <widget class="QDialog" name="DaysCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>578</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(136, 138, 133);}

QPushButton
{
	 background-color: yellow;
	border: 1px solid white;
	 border-radius: 4px;
	text-align: middle;
}

QDateEdit
{
	height:35px;
	 border-width: 1px;
      border-style: solid;
      border-radius: 4px;
}</string>
  </property>
  <widget class="QLabel" name="label_daysCalculator">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalDays">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_text">
    <item>
     <widget class="QLabel" name="label_initialDate">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_finalDate">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>91</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_date">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_initialDate">
      <item>
       <widget class="QDateEdit" name="dateEdit_initialDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_finalDate">
      <item>
       <widget class="QDateEdit" name="dateEdit_finalDate"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
